### PR TITLE
data:anuj-kumar-aj

### PR DESCRIPTION
**Pull request by Anuj-Kumar-AJ**
## Check List (Check all the applicable boxes) 

- [ X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X ] All new and existing tests passed.
- [ X] This PR does not contain plagiarized content.
- [X ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3657"></a>

### CODE REVIEW

File: data/Anuj-Kumar-AJ.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      {
+        "name": "Mastodon",
+        "url": "https://cloud-native.social/@AnujAJ",
+        "icon": "FaMastodon",
+        "color": "#00ACEE"
+      },
+      {
+        "name": "HashNode",
+        "url": "https://hashnode.com/@Anuj-kumar-aj",
+        "icon": "FaHashNode",
```
Comment (ChinmayMhatre) [inline data/Anuj-Kumar-AJ.json:38]:
```suggestion
        "icon": "SiHashnode",
```
Please use this icon

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

**Comment by Anuj-Kumar-AJ** [comment]
thanks @ChinmayMhatre  for pointing out issue

File: data/Anuj-Kumar-AJ.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        "date": "June 2022",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Submitted my First PR in eddieHub Community. Added name to member file"
+      },
+  
+      {
+        "title": "Started Learning Docker",
+        "date": "October 2022",
+        "icon": "dollar",
```
Comment (ChinmayMhatre) [inline data/Anuj-Kumar-AJ.json:63]:
```suggestion
        "icon": "FaDollarSign",
```
Please use this icon

File: data/Anuj-Kumar-AJ.json [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        "title": "Started Learning Web Development",
+        "date": "June 2022",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Started Learning ReactJS"
+      },
+      {
+        "title": "Started my OpenSource Journey",
+        "date": "June 2022",
+        "icon": "dollar",
```
Comment (ChinmayMhatre) [inline data/Anuj-Kumar-AJ.json:55]:
```suggestion
        "icon": "FaDollarSign",
```

File: data/Anuj-Kumar-AJ.json [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+        "date": "October 2022",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Started Learning Docker, for containerise application , learned about containers , images , how to pull image from dockerHub, played with images and containers in terminal"
+      },
+  
+      {
+        "title": "Started Learning Kubernetes",
+        "date": "November 2022",
+        "icon": "dollar",
```
Comment (ChinmayMhatre) [inline data/Anuj-Kumar-AJ.json:71]:
```suggestion
        "icon": "FaDollarSign",
```

File: data/Anuj-Kumar-AJ.json [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+        "color": "grey",
+        "description": "Started Learning Kubernetes, learnt about why we needed k8s,  different components of it like pods, replica, replicasets, probes,etc about its usage , installed minikube for it."
+      },
+  
+      {
+        "title": "Officially learning infrastructure and cloud",
+        "date": "December 2022",
+        "icon": "dollar",
```
Comment (ChinmayMhatre) [inline data/Anuj-Kumar-AJ.json:79]:
```suggestion
        "icon": "FaDollarSign",
```

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
A few icon changes

**Comment by ChinmayMhatre** [comment]
Hey @Anuj-Kumar-AJ , I am actually writing a post regarding reviewing GitHub issues. Would you be okay with me using this issue as an example in the post?

File: data/Anuj-Kumar-AJ.json (hunk 5 above)
Comment (ChinmayMhatre) [reply data/Anuj-Kumar-AJ.json:79]: > Suggested change
>         "icon": "dollar",
>         "icon": "FaDollarSign",

Could you send me a screenshot of the message I replied to from your end before you commit the changes? I'd really appreciate it. Thanks

**Comment by SaraJaoude** [comment]
I have committed the changes suggested by @ChinmayMhatre as a non-existent icon was used. I will now be merging this PR. 

Feel free to send any edits in a new PR.  You may wish to take a look at: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Pull request merged**